app/models/pawn.rb

```diff
@@ -11,6 +11,23 @@ def valid_move?(x_end, y_end)
 
   private
 
+  def diagonal_capture?(x_end, y_end)
+    return true if color == 'white' && capturable_black?(x_end, y_end)
+    color == 'black' && capturable_white?(x_end, y_end)
+  end
+
+  def capturable_black?(x_end, y_end)
+    return true if game.pieces.where(horizontal_position: x_end, vertical_position: y_end, color: 'black').present?
+    return true if game.pieces.where(horizontal_position: x_end, vertical_position: vertical_position, color: 'black', passant: true).present?
+    false
+  end
+
+  def capturable_white?(x_end, y_end)
+    return true if game.pieces.where(horizontal_position: x_end, vertical_position: y_end, color: 'white').present?
+    return true if game.pieces.where(horizontal_position: x_end, vertical_position: vertical_position, color: 'white', passant: true).present?
+    false
+  end
```

Comment:
`capturable_black?` and `capturable_white?` could be just 1 method passing in the color as a param:
`def capturable?(x_end, y_end, color)`